docs/release-notes/v12.3.2.md

```diff
@@ -0,0 +1,19 @@
+---
+id: v12.3.2
+title: Version 12.3.2 Release Notes
+sidebar_label: v12.3.2
+image: "img/release-note-link-preview.png"
+---
+
+_April 23th, 2024_
+
+## Fast Data
+
+### Bug Fix
+
+This version addressed the following bug:  
+* When using custom [cast functions](/fast_data/configuration/projection_storer_config.md#cast-functions-and-additional-cast-functions) in your Fast Data Project, the configuration for the Projection Storer now contains all the custom cast functions instead of just the ones from the first projection associated to the service.
```

Comment:
```suggestion
* When using custom [cast functions](/fast_data/configuration/projection_storer.md#cast-functions-and-additional-cast-functions) in your Fast Data Project, the configuration for the Projection Storer now contains all the custom cast functions instead of just the ones from the first projection associated to the service.
```